packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -35,6 +38,21 @@ class TestServiceExtensionsBinding extends BindingBase
     extensions[name] = callback;
   }
 
+  @override
+  void postEvent(String eventKind, Map<dynamic, dynamic> eventData) {
+    getEventsDispatched(eventKind).add(eventData);
+  }
+
+  List<Map<String, dynamic>> getEventsDispatched(String eventKind) {
+    return eventsDispatched.putIfAbsent(eventKind, () => <Map<String, dynamic>>[]);
+  }
+
+  Iterable<Map<String, dynamic>> getServiceExtensionChangedEvents(String extensionName) {
+    return getEventsDispatched('Flutter.ServiceExtensionChanged')
+        .where((Map<String, dynamic> event) => event['extension'] == extensionName
+    );
```

Comment:
this closing paren needs to go on the previous line.
Flutter also allows indent of just 2 spaces if you want.

Reply:
try to be consistent with the rest of the file.

Reply:
done.